### PR TITLE
Create scalatest runner for Scala 2.11

### PR DESCRIPTION
**Pull request by BeLi4L**

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Copy answer code
+cp -a /project/answer/* /project/target/source/
+
+# Find the existing fat jar
+base_fatjar=`find /project/target/source/target -name "*.jar"`
```
Comment (fraisse) [inline entrypoint.sh]: How are you sure there is a fat jar in the target directory?

File: entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+# Compile the user's code
+mkdir -p /project/target/compiled_answer
+find /project/answer -name *.scala \
+	| xargs scalac -classpath "$fatjar" -d /project/target/compiled_answer
+
+# Add it to the fat jar
+cd /project/target/compiled_answer
+find . -name *.class \
+	| xargs jar uf "$fatjar"
```
Comment (fraisse) [inline entrypoint.sh]: You really need to modify the jar itself? You cannot add the classes in the classpath?

File: entrypoint.sh [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+# Run the given test(s)
+test_classes_dir=`dirname "$base_fatjar"`/test-classes
```
Comment (fraisse) [inline entrypoint.sh]: Why do you "introspect" the directory? Isn't it `/project/target/source/target` where you had looked for the fat jar?

File: README.md [hunk 4]
```diff
@@ -1 +1,53 @@
...
+This is a Scala 2.11 runner that executes tests thanks to scalatest.
+
+## How to Use
+
+In order to use this runner for your project, edit the `codingame.yml` file and add the following lines to your project:
+
+```yaml
+    runner:
+      name: codingame/scalatest-runner
+      version: 1-scala2.11
```
Comment (fraisse) [inline README.md]: 1-scala`-`2.11?

File: entrypoint.sh (hunk 1 above)
Comment (BeLi4L) [inline entrypoint.sh]: If the project's been built with `sbt-builder`, there's gonna be a fat jar ;)

I could add a "Constraints" paragraph in the README if you want

File: entrypoint.sh (hunk 2 above)
Comment (BeLi4L) [inline entrypoint.sh]: The classes we just compiled already exist in the fat jar, I though it'd be cleaner to overwrite them. 

I'll test if it works without this jar update ;)

File: entrypoint.sh (hunk 3 above)
Comment (BeLi4L) [inline entrypoint.sh]: No, it's inside the `scala-<version>` directory

File: README.md (hunk 4 above)
Comment (BeLi4L) [inline README.md]: Fixed ;)

File: entrypoint.sh [hunk 5]
```diff
@@ -1,24 +1,20 @@
...
+mkdir -p "$compiled_answer_dir"
+find "$answer_dir" -name *.scala \
+	| xargs scalac -classpath "$classes_dir" -d "$compiled_answer_dir"
 
-# Add it to the fat jar
-cd /project/target/compiled_answer
-find . -name *.class \
-	| xargs jar uf "$fatjar"
+# Find dependencies
+lib_jars=`find /project/target/.ivy2 -name "*.jar" | tr '\n' ':'`
```
Comment (fraisse) [inline entrypoint.sh]: I don't understand what jar(s) you need in this `.ivy` directory?

**Pull request merged**